openedx_events/learning/signals.py

```diff
@@ -7,3 +7,84 @@
 They also must comply with the payload definition specified in
 docs/decisions/0003-events-payload.rst
 """
+
+from openedx_events.learning.data import (
+    CertificateData,
+    CohortData,
+    CourseEnrollmentData,
+    RegistrationFormData,
+    StudentData,
+)
+from openedx_events.tooling import OpenEdxPublicSignal
+
+STUDENT_REGISTRATION_COMPLETED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.student.registration.completed.v1",
+    data={
+        "user": StudentData,
+        "registration_form": RegistrationFormData,
+    }
+)
+
+
+SESSION_LOGIN_COMPLETED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.auth.session.login.completed.v1",
+    data={
+        "user": StudentData,
+    }
+)
+
+
+COURSE_ENROLLMENT_CREATED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.course.enrollment.created.v1",
+    data={
+        "enrollment": CourseEnrollmentData,
+    }
+)
+
+
+COURSE_ENROLLMENT_CHANGED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.course.enrollment.changed.v1",
+    data={
+        "enrollment": CourseEnrollmentData,
+    }
+)
+
+
+COURSE_ENROLLMENT_DEACTIVATED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.course.enrollment.deactivated.v1",
+    data={
+        "enrollment": CourseEnrollmentData,
+    }
+)
```

Comment:
This framing seems a bit model-centric (as opposed to event). vs. calling this something like `UNENROLLED`.